tools/sdk-cli/src/commands.rs

```diff
@@ -205,6 +205,13 @@ pub(crate) enum Commands {
     /// The up to date node information
     NodeInfo {},
 
+    /// Configurate the node
+    ConfigureNode {
+        // Optional address to send funds to during a channel close
```

Comment:
I think it's good to identify this is for mutual closes. At the moment most closes are probably force closes, and the address will not be used there.

```suggestion
        // Optional address to send funds to during a mutual channel close
```

Reply:
Done [fb7e777](https://github.com/breez/breez-sdk/pull/765/commits/fb7e777d8aaa9b7783d038baf759200c44256a81)